docs/content/https/tls.md

```diff
@@ -316,7 +316,7 @@ spec:
 
 ### Strict SNI Checking
 
-With strict SNI checking, Traefik won't allow connections from clients connections
+With strict SNI checking enabled, Traefik won't allow connections from clients connections
```

Comment:
suggest removing the last connections in this line

Reply:
Agree with your suggestion, made the change